CHANGELOG.md

```diff
@@ -1 +1,30 @@
+# 0.6.0
+
+## Prelude
+
+Added new features and commands, such as:
+
+- 'subgit delete'
+- 'subgit import'
+- 'subgit reset'
+- 'subgit clean'
+
+## General fixes
+
+- Added error handling for 'subgit pull'.
```

Comment:
Is it `Added` or `Improved`? was it there previous handling that we made better or not

Reply:
I added a check before the main pull logic that wasn't there before